app/build.gradle

```diff
@@ -142,6 +142,7 @@ dependencies {
     implementation "com.android.support:support-annotations:$rootProject.supportLibraryVersion"
     implementation 'com.android.support.constraint:constraint-layout:1.0.2'
     compile "com.android.support:multidex:$rootProject.multidexVersion"
+    compile 'com.github.dmytrodanylyk.android-process-button:library:1.0.4'
```

Comment:
Can you change this so that the version is more consistent with the rest of the code? The version number should go in the root build.gradle

Reply:
No, I mean consistent as in put it in with the other versions, aka $rootProject.multidexVersion for example.